src/components/AssignRole/AssignRole.tsx

```diff
@@ -0,0 +1,47 @@
+import Modal from '@/components/Modal/Modal'
+import { useModal } from '@/contexts/ModalContext'
+import { Button } from '@/components/Button/Button'
+
+import combineClasses from '@/utils/combineClasses'
+import styles from './AssignRole.module.scss'
+import modalStyles from '@/components/Modal/Modal.module.scss'
+import { Inter, IBM_Plex_Sans } from 'next/font/google'
+
+const ibm = IBM_Plex_Sans({ subsets: ['latin'], weight: ['400'] })
+const inter = Inter({ subsets: ['latin'], weight: ['500'] })
```

Comment:
**nitpick:** Why do we have to duplicate font family declaration each time? I think it should be handled from one place